jupyterhub/files/hub/jupyterhub_config.py

```diff
@@ -62,6 +62,7 @@ def camelCaseify(s):
     ('active_server_limit', None),
     # base url prefix
     ('base_url', None),
+    ('cookie_secret', None),
```

Comment:
I don't think we can treat `cookie_secret` quite as transparently as the rest of the traits; I think it needs special handling because `JupyterHub.cookie_secret` is a `Bytes` trait, which should fail when you try to set it with text from yaml.

Instead, adding the same encoding from yaml text as is [currently done](https://github.com/jupyterhub/jupyterhub/blob/3663d7c8fcba2118dd469a8a4cca540cc63a39cf/jupyterhub/app.py#L1345-L1346) when it's set via env:

```python
from binascii import a2b_hex
cookie_secret_hex = get_config("hub.cookieSecret")
if cookie_secret_hex:
    c.JupyterHub.cookie_secret = a2b_hex(cookie_secret_hex)
```

Reply:
Thanks for the review and nice catching this @minrk! I've updated the PR with a commit reflecting this insight!

Reply:
@minrk, does this look good? https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/1682/commits/9aff240572627488b7e0501bfbbcea1d25757c88

Reply:
I'm marking this as resolved as I think I got it right.